docs/design/rule-manager/sequence-diagram.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
+title Rule manager sequence diagram
+
+participant FlowHS
+participant SwitchManager
+database Database
+participant RuleManager
+
+FlowHS -> Database : get switches&flows
```

Comment:
is it really get?

Reply:
I mean `get required data from database` here. Do you think I need to rephrase this?